MediaGalleryUi/view/adminhtml/web/js/image/image-edit.js

```diff
@@ -20,10 +21,42 @@ define([
             images: [],
             image: null,
             modules: {
-                mediaGridMessages: '${ $.mediaGridMessages }'
+                mediaGridMessages: '${ $.mediaGridMessages }',
+                select: '${ $.name }_select'
+            },
+            viewConfig: [
+                {
+                    component: 'Magento_MediaGalleryUi/js/image/edit/keyword-ui-select',
+                    name: '${ $.name }_select'
```

Comment:
I'd use a more specific name for the keywords field/select
```suggestion
                    name: '${ $.name }_keywords'
```